components/molecule/rating/src/Icons/starFilled.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+const StarFilled = () => (
+  <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+    <path
+      d="M12 17.27L18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
+      fill="#4790bd"
```

Comment:
this shouldn't be here

Reply:
Oks. I've removed all color for the default icons used in the component